package.json

```diff
@@ -22,18 +22,17 @@
         "grunt-contrib-connect": "~1.0.0",
         "grunt-contrib-jshint": "~1.0.0",
         "grunt-contrib-watch": "^1.1.0",
-        "grunt-mocha-phantomjs": "~4.0.0",
         "http-server": "^0.12.3",
         "jshint": "~2.8.0",
         "less": "~1.7.0",
         "mocha": "^7.2.0",
-        "mocha-phantomjs-core": "~2.0.1",
-        "phantomjs-prebuilt": "^2.1.16",
+        "mocha-headless-chrome": "^3.1.0",
         "requirejs": "^2.3.6"
     },
     "main": "src/main.js",
     "scripts": {
         "install": "bower install",
-        "test": "phantomjs node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js 'built.html' spec '{\"bail\":false}' && jshint src/*.js && jshint -c tests/.jshintrc tests/*.js"
+        "testserver": "`npm bin`/http-server -p 8088",
+        "test": "node_modules/mocha-headless-chrome/bin/start -f http://localhost:8088 -t 500000"
```

Comment:
I'm curious about `-t 500000`. I assume that's a timeout. Is that for each individual test, or the entire test run? Do any tests take anywhere near that long? Previously the timeout was something like 2s/test (with specific exceptions for known outliers), which was nice because it highlighted long-running tests quickly. I'd prefer to keep the per-test timeout low if possible.

Reply:
this is for the entire test run because it treats all of index.html as a single test.

Reply:
Interesting. Are there still time limits on individual tests?

Reply:
Previously `built.html` caused this to run against the built version of vellum

https://github.com/dimagi/Vellum/blob/31900bc23dd859e8eae784116971066808646b0a/built.html#L15

and `npm test` is was (and still is) run by Travis via `make test` after the build:

https://github.com/dimagi/Vellum/blob/31900bc23dd859e8eae784116971066808646b0a/Makefile#L37-L42

Does `useBuilt` still get set somewhere during the travis test run?